var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -24,21 +24,29 @@
 ##############################################################################
 from spack import *
 
-
```

Comment:
The Flake8 tests are crashing because you deleted this empty line. There should be 2 blank lines after an import statement.